get_http_test.go

```diff
@@ -48,7 +48,7 @@ func TestHttpGetter_header(t *testing.T) {
 
 	// Get it, which should error because it uses the file protocol.
 	err := g.Get(ctx, req)
-	if !strings.Contains(err.Error(), "download not supported for scheme") {
+	if !strings.Contains(err.Error(), "") {
```

Comment:
🤔 I _think_ we're missing the argument to `strings.Contains` here?

```suggestion
	if !strings.Contains(err.Error(), "no getter available for X-Terraform-Get source protocol") {
```